src/definitions/modules/dimmer.less

```diff
@@ -151,6 +151,9 @@
     transform-style: @transformStyle;
     perspective: @perspective;
     transform-origin: center center;
+    &.modals {
+      -moz-perspective: none;
```

Comment:
If `@perspective` had been added to resolve firefox-specific issue, why not settings `@perspective` to `none` for all browsers ?

Reply:
That's what I also thought at first, but that might break existing code, because I am not 100% sure this setting was really only implemented because of Firefox issues.
I believe the perspective setting was done for performance reasons too whenever other transforms within the page dimmer occur, like transitions. To be in the safe side (at least within 2.8.x) I decided for that extra setting for modals only. Removing the perspective setting at all, as I stated in the issue text, might lead into unknow visual/performance breaking code.
When looking at the git history, the perspective setting was needed because transition animations were not working in Firefox at all too, so removing it entirely is risky for the 2.8 branch.
We could either postpone this pr to 2.9 and remove the setting entirely, but this would most probably need lots of testing, because, as said, withing a page dimmer (which can also be used without modals!) anything can exists.

Reply:
Thanks for detail.
Thats sounds fair reason for me!